packages/playground-examples/copy/ja/3-8/Breaking Changes/Checking Unions with Index Signatures.ts

```diff
@@ -1,31 +1,32 @@
 //// { compiler: { ts: "3.8.3" } }
-// In previous versions of TypeScript, the checker would not
-// verify that undeclared fields in a union conform to any indexed
-// types in the union.
+// 以前のバージョンのTypeScriptでは、
+// インデックスシグネチャを含む共用体に宣言されていないフィールドについて
+// 型チェックが行われませんでした
 
-// You can learn about indexed types here: example:indexed-types
+// インデックスシグネチャについてはこちら: example:indexed-types
+// を参照してください。
```

Comment:
行数がずれない方が良さそうなのでこちらでお願いします :pray:

```suggestion
// インデックスシグネチャについてはこちら: example:indexed-types 
```